### PR TITLE
Charlesemrich

### PR DESCRIPTION
**Pull request by CharlesEmrich**

### CODE REVIEW

**Comment by CharlesEmrich** [comment]
Huh. Not sure why those checks are failing. In fairness, I also don't have a tests file? So maybe that's the issue.

File: server.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  //Welcome.
+  client.write('Welcome to CharlesChat, ' + client.name + '\n');
+
+  client.on('data', data => {
+    //add current character to message string
+    client.message += data;
+    //Check whether they've entered CRLF and if so, publishToAll.
+    if (data === '\r\n') {
+      room.send(client, client.message);
```
Comment (goodwid) [inline server.js]: Workaround for windows' funky telnet client breaks everyone else..

File: server.old.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+const net = require('net');
```
Comment (goodwid) [inline server.old.js]: Use the version control for this.. this .old.js file should be in a different commit or a different branch, not here alongside other working code.
